Makefile

```diff
@@ -270,12 +275,15 @@ bin/calico-felix: $(GO_FILES) \
 	# freshness checking for us.
 	$(MAKE) calico-build/golang
 	mkdir -p bin
+	mkdir -p .go-pkg-cache
+	@echo Building felix...
 	$(DOCKER_RUN_RM) \
 	    -v $${PWD}:/go/src/github.com/projectcalico/felix:rw \
+	    -v $${PWD}/.go-pkg-cache:/go/pkg/:rw \
 	    calico-build/golang \
-	    go build -o $@ $(LDFLAGS) "./go/felix/felix.go"
-	# Check that the executable is correctly statically linked.
-	ldd bin/calico-felix | grep -q "not a dynamic executable"
+	    sh -c 'go build -i -o $@ -v $(LDFLAGS) "github.com/projectcalico/felix/go/felix" && \
+               ( ldd bin/calico-felix | grep -q "not a dynamic executable" || \
+	             ( echo "Error: bin/calico-felix was not statically linked"; false ) )'
```

Comment:
Do we really need this ldd test?  Is the static linking something that could easily regress and (otherwise) be missed?